### PR TITLE
2024 Year in Review

### PR DESCRIPTION
**Pull request by SaraDPH**
ETA: Dec 20

### CODE REVIEW

File: content/blog/year-in-review-2024/index.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+# The draft setting determines whether a post is published. Set it to true if
+# you want to be able to merge the post without publishing it.
+draft: false
+
+# Use the meta_desc property to provide a brief summary (one or two sentences)
+# of the content of the post, which is useful for targeting search results or
+# social-media previews. This field is required or the build will fail the
+# linter test. Max length is 160 characters.
+meta_desc:
```
Comment (SaraDPH) [inline content/blog/year-in-review-2024/index.md:20]:
```suggestion
meta_desc: year in review 2024
```

File: content/blog/year-in-review-2024/index.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
+---
+title: "2024 Year in Review: Growth, Innovation and Appreciation"
```
Comment (SaraDPH) [inline content/blog/year-in-review-2024/index.md:2]:
```suggestion
title: "2024 Year in Review: Growth, Innovation and Appreciation"
allow_long_title: true
```

File: content/blog/year-in-review-2024/index.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+- **[Pulumi ESC](https://www.pulumi.com/docs/esc/)**: Centralized secrets management and orchestration that scales. General availability this year brought features like SDKs, versioning, tagging, and integrations with 1Password and Kubernetes secrets operators.
+- **[Pulumi Insights](https://www.pulumi.com/docs/pulumi-cloud/insights/)**: Visibility, intelligence, and control over all infrastructure resources. Insights 2.0 expanded support for resources created outside Pulumi IaC, delivering advanced graph visualizations and policy enforcement powered by Pulumi CrossGuard.
+
+{{< youtube "AepHQaXeNX0?rel=0" >}}
+
+Three core capabilities unite these products:
+
+- **[Pulumi Copilot](https://www.pulumi.com/docs/pulumi-cloud/copilot/)**: Generative AI is used to manage cloud resources.
+- **[Pulumi CrossGuard](https://www.pulumi.com/docs/iac/using-pulumi/crossguard/)**: A policy-as-code engine for compliance and best practices.
+- **[Pulumi Deployments](https://www.pulumi.com/docs/pulumi-cloud/deployments/)**: Infrastructure task orchestration.
```
Comment (SaraDPH) [inline content/blog/year-in-review-2024/index.md:80]:
```suggestion
- **[Pulumi Deployments](https://www.pulumi.com/docs/pulumi-cloud/deployments/)**: Infrastructure task orchestration.

Over 3,000 customers, including Nvidia, BMW, Unity Games, and Tivity Health, leveraged these solutions to enhance velocity, save costs, and secure their infrastructure.
```

File: content/blog/year-in-review-2024/index.md [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+# Use the meta_desc property to provide a brief summary (one or two sentences)
+# of the content of the post, which is useful for targeting search results or
+# social-media previews. This field is required or the build will fail the
+# linter test. Max length is 160 characters.
+meta_desc: "Pulumi 2024: Innovation, growth, and community. Discover ESC, Insights 2.0, Copilot, Kubernetes Operator, Drift Detection, and more from a transformative year!"
+
+# The meta_image appears in social-media previews and on the blog home page. A
+# placeholder image representing the recommended format, dimensions and aspect
+# ratio has been provided for you.
+meta_image: meta.png
```
Comment (SaraDPH) [inline content/blog/year-in-review-2024/index.md:26]:
```suggestion
meta_image: 2024-year-in-review.png
```

File: content/blog/year-in-review-2024/index.md [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+
+## Community Achievements
+
+![Placeholder Image](community-highlights.png)
+
+Pulumi's vibrant community has been at the heart of everything we've accomplished:
+
+- **Global Pulumi User Groups (Meetups)**: With 20 groups across 9 countries, including new meetups in Chicago, Columbus, São Paulo, and Tel Aviv, our community continues to grow. These meetups collectively host over 6,396 members.
+- **PulumiUP 2024**: Over 8,000 registrants globally, with nearly 1,000 for the first-ever PulumiUP Europe.
+- **Workshops**: 72 workshops with 14,500+ registrations to learn Pulumi from intro to advanced topics
```
Comment (SaraDPH) [inline content/blog/year-in-review-2024/index.md:123]:
```suggestion
- **Workshops**: 72 workshops with 14,500+ registrations from engineers wanting to learn Pulumi.

File: content/blog/year-in-review-2024/index.md [hunk 6]
```diff
@@ -0,0 +1,132 @@
...
+
+![Placeholder Image](3000-customers.png)
+
+Pulumi's growth and impact this year have been amazing:
+
+- **75,000+ Pull Requests**: Driven by 4,400+ contributors.
+- **3,000+ Customers**: Empowering industries like AI, gaming, finance, and healthcare.
+- **100M+ Downloads**: Pulumi IaC adoption soared globally.
+- **167% Contribution Growth**: Outpacing other IaC tools like Terraform and OpenTofu.
+- **71% Cloud Cost Savings**: Customers like Tivity Health saved millions with Pulumi.
```
Comment (SaraDPH) [inline content/blog/year-in-review-2024/index.md:113]:
```suggestion
- **75,000+ Pull Requests**: Driven by 5,600+ contributors.
- **3,000+ Customers**: Empowering industries like AI, gaming, finance, and healthcare.
- **100M+ Downloads**: Pulumi IaC adoption soared globally.
- **167% Contribution Growth**: Outpacing other IaC tools like Terraform and OpenTofu.
- **71% Cloud Cost Savings**: Customers like Tivity Health saved millions with Pulumi.
```

File: content/blog/year-in-review-2024/index.md [hunk 7]
```diff
@@ -0,0 +1,132 @@
...
+
+Here's a look at how Pulumi has grown, evolved, and continued empowering teams worldwide.
+
+<!--more-->
+
+## A Unified Vision: The Pulumi Platform
+
+![Placeholder Image](pulumi-platform-vision.png)
+
+This year, Pulumi unveiled a bold new vision for Pulumi—[a comprehensive product suite](https://www.pulumi.com/blog/pulumi-up-2024/) that extends far beyond infrastructure as code.. The Pulumi platform now consists of three core products:
```
Comment (thtmnisamnstr) [inline content/blog/year-in-review-2024/index.md:70]:
```suggestion
This year, Pulumi unveiled a bold new vision for Pulumi—[a comprehensive product suite](https://www.pulumi.com/blog/pulumi-up-2024/) that extends far beyond infrastructure as code. The Pulumi platform now consists of three core products:
```

File: content/blog/year-in-review-2024/index.md [hunk 8]
```diff
@@ -0,0 +1,132 @@
...
+
+- **[Pulumi Copilot](https://www.pulumi.com/docs/pulumi-cloud/copilot/)**: Generative AI is used to manage cloud resources.
+- **[Pulumi CrossGuard](https://www.pulumi.com/docs/iac/using-pulumi/crossguard/)**: A policy-as-code engine for compliance and best practices.
+- **[Pulumi Deployments](https://www.pulumi.com/docs/pulumi-cloud/deployments/)**: Infrastructure task orchestration.
+
+Over 3,000 customers, including Nvidia, BMW, Unity Games, and Tivity Health, leveraged these solutions to enhance velocity, save costs, and secure their infrastructure.
+
+## Highlights Across the Product Innovation
+
+![Placeholder Image](pulumi-platform-vision.png)
```
Comment (thtmnisamnstr) [inline content/blog/year-in-review-2024/index.md:88]: @SaraDPH Probably remove this. It's used earlier in the post.

File: content/blog/year-in-review-2024/index.md [hunk 9]
```diff
@@ -0,0 +1,132 @@
...
+
+Three core capabilities unite these products:
+
+- **[Pulumi Copilot](https://www.pulumi.com/docs/pulumi-cloud/copilot/)**: Generative AI is used to manage cloud resources.
+- **[Pulumi CrossGuard](https://www.pulumi.com/docs/iac/using-pulumi/crossguard/)**: A policy-as-code engine for compliance and best practices.
+- **[Pulumi Deployments](https://www.pulumi.com/docs/pulumi-cloud/deployments/)**: Infrastructure task orchestration.
+
+Over 3,000 customers, including Nvidia, BMW, Unity Games, and Tivity Health, leveraged these solutions to enhance velocity, save costs, and secure their infrastructure.
+
+## Highlights Across the Product Innovation
```
Comment (thtmnisamnstr) [inline content/blog/year-in-review-2024/index.md:86]: This doesn't make sense. Did you mean "Highlights Across the Product"?

File: content/blog/year-in-review-2024/index.md [hunk 10]
```diff
@@ -0,0 +1,132 @@
...
+
+![Placeholder Image](pulumi-platform-vision.png)
+
+Pulumi's product innovations this year have empowered teams to automate, secure, and manage cloud infrastructure with confidence:
+
+- **[Pulumi ESC General Availability](https://www.pulumi.com/blog/pulumi-esc-ga/)**, redefining secrets management with features like dynamic credentials and hierarchical environments. New capabilities include secrets syncing with external stores like AWS Secrets Manager and Kubernetes operators for runtime integration.
+- **[Pulumi Insights 2.0](https://www.pulumi.com/blog/pulumi-insights-2/)** delivers AI-powered search, a comprehensive inventory for resources created by other tools like Terraform and CloudFormation, and automated remediation capabilities powered by CrossGuard.
```
Comment (thtmnisamnstr) [inline content/blog/year-in-review-2024/index.md:93]:
```suggestion
- **[Pulumi Insights 2.0](https://www.pulumi.com/blog/pulumi-insights-2/)** delivers AI-powered search, a comprehensive inventory for resources created by Pulumi IaC as well as other tools like Terraform and CloudFormation, and automated remediation capabilities powered by CrossGuard.
```

File: content/blog/year-in-review-2024/index.md [hunk 11]
```diff
@@ -0,0 +1,132 @@
...
+- **167% Contribution Growth**: Outpacing other IaC tools like Terraform and OpenTofu.
+- **71% Cloud Cost Savings**: Customers like Tivity Health saved millions with Pulumi.
+
+## Community Achievements
+
+![Placeholder Image](community-highlights.png)
+
+Pulumi's vibrant community has been at the heart of everything we've accomplished:
+
+- **Global Pulumi User Groups (Meetups)**: With 20 groups across 9 countries, including new meetups in Chicago, Columbus, São Paulo, and Tel Aviv, our community continues to grow. These meetups collectively host over 6,396 members.
```
Comment (thtmnisamnstr) [inline content/blog/year-in-review-2024/index.md:121]:
```suggestion
- **Global Pulumi User Groups (Meetups)**: With 20 groups across 9 countries, including new meetups in Chicago, Columbus, São Paulo, and Tel Aviv, our community continues to grow. These meetups collectively hosted over 6,396 members.
```

**Review by thtmnisamnstr — APPROVED** [review]
I left a few suggestions but good otherwise.

File: content/blog/year-in-review-2024/index.md (hunk 9 above)
Comment (SaraDPH) [inline content/blog/year-in-review-2024/index.md:86]:
```suggestion
## Highlights of Releases and Enhancements 
```

Comment (SaraDPH) [reply content/blog/year-in-review-2024/index.md:86]: Highlights of Releases and Enhancements?

File: content/blog/year-in-review-2024/index.md [hunk 12]
```diff
@@ -0,0 +1,132 @@
...
+- **167% Contribution Growth**: Outpacing other IaC tools like Terraform and OpenTofu.
+- **71% Cloud Cost Savings**: Customers like Tivity Health saved millions with Pulumi.
+
+## Community Achievements
+
+![Placeholder Image](community-highlights.png)
+
+Pulumi's vibrant community has been at the heart of everything we've accomplished:
+
+- **Global Pulumi User Groups (Meetups)**: With 20 groups across 9 countries, including new meetups in Chicago, Columbus, São Paulo, and Tel Aviv, our community continues to grow. These meetups collectively hosted over 6,396 members.
```
Comment (SaraDPH) [inline content/blog/year-in-review-2024/index.md:121]:
```suggestion
- **Global [Pulumi User Groups](https://www.meetup.com/pro/pugs/) (PUGs)**: With 20 groups across 9 countries, including new meetups in [Chicago](https://www.meetup.com/chicago-pulumi-user-group/), [Columbus](https://www.meetup.com/columbus-pulumi-user-group), [São Paulo](https://www.meetup.com/sao-paulo-pulumi-user-group), and [Tel Aviv](https://www.meetup.com/tel-aviv-pulumi-user-group), our community continues to grow. These meetups collectively hosted over 6,396 members.
```

File: content/blog/year-in-review-2024/index.md [hunk 13]
```diff
@@ -0,0 +1,132 @@
...
+- **PulumiUP 2024**: Over 8,000 registrants globally, with nearly 1,000 for the first-ever PulumiUP Europe.
+- **Workshops**: 72 workshops with 14,500+ registrations from engineers wanting to learn Pulumi.
+- **22,000 GitHub Stars**: A testament to our community's passion and dedication.
```
Comment (SaraDPH) [inline content/blog/year-in-review-2024/index.md:124]:
```suggestion
- **[Workshops](https://www.pulumi.com/resources/#upcoming)**: 72 workshops with 14,500+ registrations from engineers wanting to learn Pulumi.
- **22,000 GitHub Stars**: A testament to our community's passion and dedication.
```

File: content/blog/year-in-review-2024/index.md [hunk 14]
```diff
@@ -0,0 +1,132 @@
...
+
+![Placeholder Image](3000-customers.png)
+
+Pulumi's growth and impact this year have been amazing:
+
+- **75,000+ Pull Requests**: Driven by 5,600+ contributors.
+- **3,000+ Customers**: Empowering industries like AI, gaming, finance, and healthcare.
+- **100M+ Downloads**: Pulumi IaC adoption soared globally.
+- **167% Contribution Growth**: Outpacing other IaC tools like Terraform and OpenTofu.
+- **71% Cloud Cost Savings**: Customers like Tivity Health saved millions with Pulumi.
```
Comment (SaraDPH) [inline content/blog/year-in-review-2024/index.md:113]:
```suggestion
- **71% Cloud Cost Savings**: Customers like [Tivity Health saved millions](https://www.pulumi.com/blog/devsecops-strategy-security-automation-tivity-health/) with Pulumi.
```

File: content/blog/pulumi-year-in-review/index.md [hunk 15]
```diff
@@ -0,0 +1,132 @@
...
+title: "2024 Year in Review: Growth, Innovation and Appreciation"
+allow_long_title: true
+
+# The date represents the post's publish date, and by default corresponds with
+# the date and time this file was generated. Dates are used for display and
+# ordering purposes only; they have no effect on whether or when a post is
+# published. To influence the ordering of posts published on the same date, use
+# the time portion of the date value; posts are sorted in descending order by
+# date/time.
+date: 2024-12-19T06:11:09Z
```
Comment (SaraDPH) [inline content/blog/pulumi-year-in-review/index.md:11]:
```suggestion
date: 2024-12-20T06:11:09Z
```

File: content/blog/pulumi-year-in-review/index.md [hunk 16]
```diff
@@ -0,0 +1,132 @@
...
+
+- **[Pulumi Copilot](https://www.pulumi.com/docs/pulumi-cloud/copilot/)**: Generative AI is used to manage cloud resources.
+- **[Pulumi CrossGuard](https://www.pulumi.com/docs/iac/using-pulumi/crossguard/)**: A policy-as-code engine for compliance and best practices.
+- **[Pulumi Deployments](https://www.pulumi.com/docs/pulumi-cloud/deployments/)**: Infrastructure task orchestration.
+
+Over 3,000 customers, including Nvidia, BMW, Unity Games, and Tivity Health, leveraged these solutions to enhance velocity, save costs, and secure their infrastructure.
+
+## Highlights of Releases and Enhancements 
+
+![Placeholder Image](pulumi-platform-vision.png)
```
Comment (SaraDPH) [inline content/blog/pulumi-year-in-review/index.md:88]:
```suggestion
![Placeholder Image](any-language-any-cloud.png)
```

**Comment by shughes26** [comment]
Verified build failure is a false positive from the link checker

**Pull request merged**